gRPC/rust-sdks/cosmwasm_client/src/cosmwasm_connector.rs

```diff
@@ -0,0 +1,74 @@
+use crate::pull_service::PullResponseCosmWasm;
+use cosmrs::cosmwasm::MsgExecuteContract;
+use cosmrs::rpc::Client;
+use cosmrs::{
+    crypto::secp256k1,
+    rpc,
+    tx::{self, AccountNumber, Fee, Msg, SignDoc, SignerInfo},
+    AccountId, Coin,
+};
+use serde_json::json;
+use std::str::FromStr;
+
+pub async fn invoke_cosmwasm_chain(cosmwasm: PullResponseCosmWasm) {
+    const CHAIN_ID: &str = "osmo-test-5";
+    const ACCOUNT_NUMBER: AccountNumber = 25903;
```

Comment:
Account number is depend on wallet address, so we can define like this `<ACCOUNT NUMBER>`